cms/static/sass/_lib.scss

```diff
@@ -0,0 +1,9 @@
+// ------------------------------
+// Open edX Certificates: Main Style Compile
+
+// About: third party libraries and dependencies import
+
+
+@import '../../../node_modules/edx-pattern-library/node_modules/bourbon/app/assets/stylesheets/bourbon';
```

Comment:
I'm worried about reaching into the Pattern Library's `node_modules` directory. I think we should encapsulate this within a pattern library SASS module.